(DOCSP-22940): Created new Realm Web SDK Install Page and Updated (#2035)
mongodb/docs-realm: source/web.txt

.. note::

   There are SDKs built for :ref:`React Native <react-native-intro>`

>    and :ref:`Node.js <node-intro>` which should be used when creating

Reviewer: > "should be used "

rephrase to avoid passive voice

   apps in these environments. Alternatively, you 
   can use the Web SDK for React Native or Node.js apps, but additional 

>    steps must be taken. For further instruction, visit the :ref:`Web SDK 

Reviewer: > but additional steps must be taken.

avoid passive voice

>    can use the Web SDK for React Native or Node.js apps, but you must take 
>    additional steps. For further instruction, visit the :ref:`Web SDK 
>    installation page <install>`.

Reviewer: ```suggestion
   can use the Web SDK for React Native or Node.js apps. For further instruction, refer to :ref:`Install for Node.js <install>`.
```
